### PR TITLE
Fix link for WES-entropy-client

### PR DESCRIPTION
**Pull request by peschmae**
The link to the `WES-entropy-client` is no longer valid, as all repositories from Whitewood Crypto are gone.

There is a fork available on github, which at least for now only contains the initial commit by whitewood, but I'm not sure if you want to trust that. So alternatively it could also be removed completely. 

This commit replaces the link, with the remaining repo, but I can also change it, to remove the link.

### CODE REVIEW

File: docs/production/index.rst [hunk 1]
```diff
@@ -49,7 +49,7 @@ The amount of effort you wish to expend ensuring that Lemur has good entropy to
 
 If you wish to generate more entropy for your system we would suggest you take a look at the following resources:
 
-- `WES-entropy-client <https://github.com/WhitewoodCrypto/WES-entropy-client>`_
+- `WES-entropy-client <https://github.com/Virginian/WES-entropy-client>`_
```
Comment (hosseinsh) [inline docs/production/index.rst:52]: maybe we should mark it as such, that this is clone of the presently deleted WhitewoodCrypto repo

Comment (peschmae) [reply docs/production/index.rst:52]: I've updated the pull request with 5d7ca85

**Comment by hosseinsh** [comment]
Thanks @peschmae for the PR, and addressing the feedback. merging shortly.

**Comment by peschmae** [comment]
Not sure why travis ci failed, last time it looked fine to me, and running the tests locally also works fine.

@hosseinsh can you trigger a rerun of the build?

**Comment by hosseinsh** [comment]
> Not sure why travis ci failed, last time it looked fine to me, and running the tests locally also works fine.
> 
> @hosseinsh can you trigger a rerun of the build?

yeah, that looked strange.

**Comment by hosseinsh** [comment]
thanks again! finally merged.

**Pull request merged**